org.gajim.Gajim_nightly.yaml

```diff
@@ -0,0 +1,331 @@
+        sha256: c69b9f633a0c03c1b84ba14c73b0ec132f6323056e675702c1a5f75f316e06fb
+
+  # GSound dependency
+  - shared-modules/libcanberra/libcanberra.json
+
+  - name: gsound
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gsound/1.0/gsound-1.0.3.tar.xz
+        sha256: ca2d039e1ebd148647017a7f548862350bc9af01986d39f10cfdc8e95f07881a
+
+  - name: gspell
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gspell/1.8/gspell-1.8.4.tar.xz
+        sha256: cf4d16a716e813449bd631405dc1001ea89537b8cdae2b8abfb3999212bd43b4
+
+  - name: farstream
+    rm-configure: true
+    sources:
+      - type: archive
+        url: https://freedesktop.org/software/farstream/releases/farstream/farstream-0.2.9.tar.gz
+        sha256: cb7d112433cf7c2e37a8ec918fb24f0ea5cb293cfa1002488e431de26482f47b
+      - type: patch
+        path: farstream-make-4.3.patch
+
+  - name: python3-pillow
+    buildsystem: simple
+    build-commands:
+      - pip3 install .
+    sources:
+      - type: archive
+        url: https://files.pythonhosted.org/packages/source/p/pillow/Pillow-9.0.1.tar.gz
+        sha256: 6c8bc8238a7dfdaf7a75f5ec5a663f4173f8c367e5a39f87e720495e1eed75fa
+
+  # sentry-sdk dependency
+  - name: python3-urllib3
+    buildsystem: simple
+    build-commands:
+      - pip3 install urllib3-1.26.9-py2.py3-none-any.whl
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/py2.py3/u/urllib3/urllib3-1.26.9-py2.py3-none-any.whl
+        sha256: 44ece4d53fb1706f667c9bd1c648f5469a2ec925fcf3a776667042d645472c14
+
+  - name: python3-certifi
+    buildsystem: simple
+    build-commands:
+      - pip3 install certifi-2021.10.8-py2.py3-none-any.whl
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/py2.py3/c/certifi/certifi-2021.10.8-py2.py3-none-any.whl
+        sha256: d62a0163eb4c2344ac042ab2bdf75399a71a2d8c7d47eac2e2ee91b9d6339569
+
+  - name: python3-sentry-sdk
+    buildsystem: simple
+    build-commands:
+      - pip3 install sentry_sdk-1.5.8-py2.py3-none-any.whl
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/py2.py3/s/sentry_sdk/sentry_sdk-1.5.8-py2.py3-none-any.whl
+        sha256: 32af1a57954576709242beb8c373b3dbde346ac6bd616921def29d68846fb8c3
+
+  - name: python3-nbxmpp
+    buildsystem: simple
+    build-commands:
+      - pip3 install .
+    sources:
+      - type: git
+        url: https://dev.gajim.org/gajim/python-nbxmpp.git
+
+  - name: gajim
+    buildsystem: simple
+    build-commands:
+      - pip3 install .
+      - touch /app/share/run-as-flatpak
+      - echo -e '#!/bin/sh\ngajim --gapplication-app-id=org.gajim.Gajim_nightly "$@"' > /app/bin/gajim-devel
+      - chmod 755 /app/bin/gajim-devel
+    sources:
+      - type: git
+        url: https://dev.gajim.org/gajim/gajim.git
```

Comment:
See:
https://github.com/flathub/flathub/issues/2975#issuecomment-1086897335

Flathub is not for nightly builds.